authors/apps/articles/models.py

```diff
@@ -56,3 +60,16 @@ class Ratings(models.Model):
 
     class Meta:
         ordering = ['-ratings', ]
+
+
+class LikesDislikesModel(models.Model):
+    user_liking = models.ForeignKey(
+        Profile, related_name='user_rating' , 
```

Comment:
This `related_name` field is misleading as there is no relation between this model and the **Ratings** model. In any case, I found it unnecessary since it is not being used anywhere in your code and the code works fine without it. Kindly remove/rename it.

Reply:
Okay kool, let me fix this

Reply:
I have fixed this